src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx

```diff
@@ -8,8 +8,6 @@ translate:
   - jp
 metaDescription: How the New Relic One pricing plan and billing calculations work.
 redirects:
-  - /docs/accounts/original-accounts-billing/product-based-pricing/original-product-based-pricing
-  - /docs/accounts/original-accounts-billing/product-pricing/product-based-pricing
```

Comment:
Removing this one makes sense, that let http://localhost:8000/docs/accounts/original-accounts-billing/product-pricing/product-based-pricing resolve normally.